mlflow/langchain/runnables.py

```diff
@@ -44,11 +45,20 @@
 
 
 def _load_model_from_config(path, model_config):
-    from langchain.chains.loading import load_chain
+    import langchain
     from langchain.chains.loading import type_to_loader_dict as chains_type_to_loader_dict
-    from langchain.llms.loading import get_type_to_cls_dict as llms_get_type_to_cls_dict
-    from langchain.llms.loading import load_llm
-    from langchain.prompts.loading import load_prompt
+
+    if Version(langchain.__version__) < Version("0.0.349"):
+        from langchain.llms.loading import get_type_to_cls_dict as llms_get_type_to_cls_dict
+    else:
+        try:
+            from langchain_community.llms import get_type_to_cls_dict as llms_get_type_to_cls_dict
```

Comment:
Can we import this from `langchain.llms`?

https://github.com/langchain-ai/langchain/blob/b9ef92f2f4bb1e4322dff040c4474c0922fbf546/libs/langchain/langchain/llms/__init__.py#L799